gazebo_plugins/scripts/gazebo_model

```diff
@@ -57,12 +56,11 @@ if __name__ == "__main__":
 
         model_msg = GazeboModel(model_name,param_name,GazeboModel.URDF_PARAM_NAME,namespace,Pose(Point(lx,ly,lz),Quaternion()))
         success = gazebo_plugins_interface.load_model(model_msg)
-        print "spawning success",success
+        print("spawning success", success)
```

Comment:
Probably a space to separate the string from the value would look nicer
```suggestion
        print("spawning success ", success)
```

Reply:
`print()` joins arguments with a space already

```python3
>>> print("foo", "bar")
foo bar
```

Reply:
4b50082